ChaosMod/Effects/db/Peds/PedsSpawnBallaSquad.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+	Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+	std::vector<int> ballaNames = { -198252413, 588969535, 361513884, -1492432238, -1410400252, 599294057 };
```

Comment:
Not really a big deal in this case, but this should be `static const` so you don't create a new list every time the effect starts